packages/strapi-plugin-uuid-field/admin/src/utils/getTrad.ts

```diff
@@ -0,0 +1,5 @@
+import pluginId from '../pluginId';
+
+const getTrad = (id: string) => `${pluginId}.${id}`;
```

Comment:
Wat betekent `getTrad`?

Reply:
Deze functie helpt bij het vertalen van de plugin.

- Hoe werkt het:
  ```ts
     getTrad(plugin-naam + bericht-id-van-de-json)
  ```
 - `getTrad` volgt de naamgevingsconventie van Strapi, waarbij 'Trad' waarschijnlijk een verkorte vorm is van 'translation' of 'traduction' (het Franse woord voor vertaling), aangezien Strapi een Frans bedrijf is. Meer informatie over Strapi is te vinden op https://strapi.io/about-us-old."